lib/sassc/engine.rb

```diff
@@ -17,6 +17,8 @@ def initialize(template, options = {})
     end
 
     def render
+      return @template if @template.empty?
+
```

Comment:
maybe that test could be adapted for taking the empty string into account, since the encoding shouldn't matter for empty strings...